pallet-chainlink-feed/src/lib.rs

```diff
@@ -38,10 +38,17 @@ pub mod pallet {
 	use sp_std::prelude::*;
 
 	use crate::{
+		default_weights::SubmitWeight,
 		traits::OnAnswerHandler,
 		utils::{median, with_transaction_result},
 	};
 
+	/// Configuration for submiting paysfee
+	pub enum SubmitterPaysFee {
```

Comment:
Add some derives?

Reply:
Just an enum...No need to derive anything and the node can compile this